common/make_migrations.sh

```diff
@@ -27,12 +27,12 @@ docker run \
        --env-file common/environments/dev \
        --interactive \
        --link drdb:postgres\
-       dr_models python3.6 manage.py makemigrations data_refinery_common --noinput
+       dr_models python3.6 manage.py makemigrations data_refinery_common
 
 docker run \
        --volume $script_directory/data_refinery_common:/home/user/data_refinery_common \
        --add-host=database:$DB_HOST_IP \
        --add-host=nomad:$HOST_IP \
        --env-file common/environments/dev \
        --link drdb:postgres \
-       dr_models python3.6 manage.py migrate --noinput
```

Comment:
Why was this removed? This is necessary for running this command in CI. That's why we added it in the first place.

Reply:
This ticket I opened with Django sums up my beef with that flag: https://code.djangoproject.com/ticket/29470